wikichron/dash/apps/monowiki/metrics/metricClasses/areaChart.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+AreaChart class.
```

Comment:
Filenames of files for classes should start with upper case.

Reply:
Done.